src/store/cart-store.js

```diff
@@ -0,0 +1,28 @@
+import { Store } from 'consus-flux';
+
+let items = [];
+
+class CartStore extends Store {
+    getItems(){
+        return items;
+    }
+}
+
+const store = new CartStore();
+
+store.registerHandler('CHECKOUT_ITEM_FOUND', data => {
+    let item = {
+        id: data.id,
+        status: data.status
+    };
+    items.push(item);
```

Comment:
On second thought, we're only worried about the happy path at the moment. This is fine :+1: